src/site/content/en/handbook/grammar/index.md

```diff
@@ -20,10 +20,16 @@ description: |
   }
 </style>
 
-Grammar and mechanics are important for making writing clear, but don't let them get in the way of putting text on the page! Content reviewers will always support you with editorial stuff. That said, here are a few conventions to be aware of.
+Grammar and mechanics are important for making writing clear. Content reviewers review posts and codelabs for editorial issues covered in this post. If a piece is already error free, the entire review process goes faster. To keep things simple, web.dev uses the following resources:
+* The [Google Developer Documentation Style Guide](https://developers.google.com/style/) (GDDSG) is the primary reference for writing style issues.
+* If a style issue isn't covered in the GDDGS, check the [Chicago Manual of Style](https://www.chicagomanualofstyle.org/home.html).
+* [Merriam-Webster](http://www.m-w.com) is the primary reference for spelling and capitalization.
+* The [Word list](/handbook/word-list) provides spelling and capitalization conventions specific to web.dev.
```

Comment:
kinda odd to link to the web.dev word list here, since this is a list of external resources web.dev follows. I would remove this bullet.

Reply:
I was thinking of this more as a prioritized list of references, and the web.dev word list is relevant because it does provide punctuation and spelling guidance for cases not covered by the other references.

Reply:
I agree. You need some place indicate such things as 'home screen' needs to be two words (as of when I last checked with Android folks), or that DevTools has to be camel case.